tests/waku_relay/test_waku_relay.nim

```diff
@@ -1,211 +0,0 @@
-{.used.}
-
-import
-  std/[options, sequtils, strutils],
```

Comment:
Why this file is being removed?

Reply:
Those were legacy tests that have been replaced for the new ones.